freqtrade/persistence/models.py

```diff
@@ -254,11 +257,20 @@ class LocalTrade():
     max_rate: float = 0.0
     # Lowest price reached
     min_rate: float = 0.0
-    sell_reason: str = ''
-    sell_order_status: str = ''
+    close_reason: str = ''   
+    close_order_status: str = '' 
     strategy: str = ''
     timeframe: Optional[int] = None
 
+    #Margin trading properties
+    leverage: Optional[float] = None
+    borrowed: float = 0
+    borrowed_currency: float = None
+    interest_rate: float = 0
+    min_stoploss: float = None
+    isShort: boolean = False
```

Comment:
If you add these here (and intend to use them) - you MUST add them as columns to the Trade object too - otherwise they won't be stored to database.

Reply:
Are you referring to the Trade class that is also in models.py? Because they are also added there. Did I miss one?

Reply:
You added it about 2 minutes after i made this comment :laughing: 

so all good now i think.